doc/shared/sundomeigest/SUNDomEigEst_Power.rst

```diff
@@ -0,0 +1,213 @@
+..
+   Programmer(s): Mustafa Aggul @ SMU
+   ----------------------------------------------------------------
+   SUNDIALS Copyright Start
+   Copyright (c) 2002-2025, Lawrence Livermore National Security
+   and Southern Methodist University.
+   All rights reserved.
+
+   See the top-level LICENSE and NOTICE files for details.
+
+   SPDX-License-Identifier: BSD-3-Clause
+   SUNDIALS Copyright End
+   ----------------------------------------------------------------
+
+.. _SUNDomEigEst.POWER:
+
+The SUNDomEigEst_Power Module
+======================================
+
+The SUNDomEigEst_Power implementation of the ``SUNDomEigEstimator`` class performs
+the Power Iteration (PI) method :cite:p:`vonmises29`; this is an iterative dominant
+eigenvalue estimator that is designed to be compatible with any ``N_Vector``
+implementation that supports a minimal subset of operations (:c:func:`N_VClone()`,
+:c:func:`N_VDotProd()`,  :c:func:`N_VScale()`, and :c:func:`N_VDestroy()`).
+
+Power iteration is useful for large, sparse matrices whose dominant eigenvalue  
+is real-valued and has algebraic multiplicity one. The algorithm starts with a non-zero 
+vector :math:`\mathbf{v}_{0}`.  It then  iteratively updates this via
+
+.. math::
+
+    \mathbf{v}_{k+1} = \frac{A \mathbf{v}_k}{\|A \mathbf{v}_k\|},
+
+where :math:`\| \cdot \|` denotes the Euclidean norm.  Over successive iterations,
+:math:`\mathbf{v}_k` converges to the eigenvector corresponding to
+the dominant eigenvalue of :math:`A`.  At each step, the corresponding eigenvalue
+can be approximated using the Rayleigh quotient
+
+.. math::
+
+    \lambda_k = \frac{\mathbf{v}_k^T A \mathbf{v}_k}{\|\mathbf{v}_k\|^2}.
+
+The iteration continues until the two successive eigenvalue approximations are
+relatively close enough to one another.  That is, for some :ref:`relative tolerance <pi_rel_tol>`.
+
+Power iteration works for the matrices that have a **real** dominant eigenvalue.  
+If it is strictly greater than all others (in magnitude), convergence is guaranteed.  
+The speed of convergence depends on the ratios of the magnitude of the first two dominant eigenvalues.
+
+The matrix :math:`A` is not required explicitly; only a routine that provides  
+the matrix-vector product :math:`Av` is required.  Also, PI requires a fixed 
+amount of memory regardless of the number of iterations.  
+
+
+.. _SUNDomEigEst.POWER.Usage:
+
+SUNDomEigEst_Power Usage
+-----------------------------
+
+The header file to be included when using this module is ``sundomeigest/sundomeigest_power.h``.
+The SUNDomEigEst_Power module is accessible from all SUNDIALS solvers *without* linking to the
+``libsundials_sundomeigestpi`` module library.
```

Comment:
Copying from previous PR:
@maggul asked: "@gardner48, is this still a true statement after your adjustments in cmakes?"